src/_translations/locales/nl.json

```
@@ -26,6 +26,18 @@
     "WHERE_TO_BUY": "Waar koop ik een UiTPAS?",
     "LOGIN": "Aanmelden"
   },
+  "PROFILE": {
+    "POINTS": "punten",
+    "HELLO": "Dag",
```

More futureproof to use interpolation here.

```suggestion
    "HELLO": "Dag {{name}}",
```

---

fixed in next PR